addons/test_mail/tests/test_mail_activity.py

```diff
@@ -225,6 +226,28 @@ def test_activity_summary_sync(self):
             # activity summary remains unchanged from change of activity type as call activity doesn't have default summary
             self.assertEqual(ActivityForm.summary, email_activity_type.summary)
 
+    @mute_logger('odoo.sql_db')
+    @users('employee')
+    def test_activity_values(self):
+        test_record = self.env['mail.test.activity'].browse(self.test_record.ids)
+
+        with self.assertRaises(IntegrityError):
+            self.env['mail.activity'].create({
+                'res_model_id': self.env['ir.model']._get_id(test_record._name),
+            })
```

Comment:
(same reason as above)
```suggestion
            'res_id': 0,
})
```

Reply:
Updated as above